config/routes.rb

```diff
@@ -1,8 +1,20 @@
 Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   root "users#index"
-
-  resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-  end
-
-end
\ No newline at end of file
+  get "users/new", to: "users#new"
+  post "users", to: "users#create"
+  get "users/:id", to: "users#show"
+  get "users/:id/edit", to: "users#edit"
+  patch "users/:id", to: "users#update"
+  delete "users/:id", to: "users#destroy"
+  
+  get "users/:id/posts", to: "posts#index", as: 'user_posts'
+  get "users/:id/posts/new", to: "posts#new", as: 'new_post'
```

Comment:
- [ ] It looks like You have not created the view for the creation of the new post. Please create one 😃 👍 
![image](https://user-images.githubusercontent.com/90222110/216767032-95c9578c-dfd9-4cd5-881a-df7f2acd5a95.png)